src/EnergyPlus/Vectors.cc

```diff
@@ -489,7 +476,8 @@ void CalcCoPlanarNess(Array1D<Vector> &Surf, int const NSides, bool &IsCoPlanar,
     if (std::abs(MaxDist) > Constant::SmallDistance) IsCoPlanar = false;
 }
 
-std::vector<int> PointsInPlane(Array1D<Vector> &BaseSurf, int const BaseSides, Array1D<Vector> &QuerySurf, int const QuerySides, bool &ErrorFound)
+std::vector<int>
+PointsInPlane(Array1D<Vector> &BaseSurf, int const BaseSides, Array1D<Vector> const &QuerySurf, int const QuerySides, bool &ErrorFound)
```

Comment:
[src/EnergyPlus/Vectors.cc:492]:(style),[constParameterReference],Parameter 'QuerySurf' can be declared as reference to const